model/user-agent/registry.yaml

```
@@ -35,3 +35,21 @@ groups:
           using a user-agent for non-browser products, such as microservices with multiple names/versions inside the
           `user_agent.original`, the most significant version SHOULD be selected. In such a scenario it should align
           with `user_agent.name`
+      - id: user_agent.synthetic.type
+        stability: experimental
+        brief: >
+         Specifies the category of synthetic traffic, such as tests or bots.
+        note: >
+          This flag can primarily be determined by the contents of the `user_agent.original` attribute. Components that populate the attribute should determine what they consider synthetic or bot traffic,
+          and set this attribute accordingly. This attribute can either be set for self-identification purposes, or on telemetry detected to be generated as a result
+          of a synthetic request. This attribute is useful for distinguishing between genuine client traffic and synthetic traffic generated by bots or tests.
```

(removed the word "should", or could change it to "SHOULD")
```suggestion
          This attribute MAY be derived from the contents of the `user_agent.original` attribute.
          Components that populate the attribute are responsible for determining what they consider to be synthetic bot or test traffic.
          This attribute can either be set for self-identification purposes, or on telemetry detected to be generated as a result
          of a synthetic request. This attribute is useful for distinguishing between genuine client traffic and synthetic traffic generated by bots or tests.
```

---

I think MAY makes sense here as it's not a requirement, but simply a possible choice.